storage/remote/queue_manager.go

```diff
@@ -115,7 +115,7 @@ func newQueueManagerMetrics(r prometheus.Registerer, rn, e string) *queueManager
 		Subsystem:   subsystem,
 		Name:        "sent_batch_duration_seconds",
 		Help:        "Duration of sample batch send calls to the remote storage.",
-		Buckets:     prometheus.DefBuckets,
+		Buckets:     prometheus.ExponentialBuckets(.03125, 2, 15),
```

Comment:
Rather than redefining several buckets that people might be using for SLIs, what would you think of `append(prometheus.DefBuckets, 30, etc...)`? It is also a bit easier to know what the range is at a glance that way.

Reply:
My original thought was to keep config simpler with using one fo the generated bucket sizes, but that wasn't terribly straightforward, so this seems reasonable.

I'm wondering if there should be a bucket at 20 or 25 instead of 30, since I don't know if a timeout would register for the 30s bucket.

Reply:
How about `25, 60, 120, 300` That keeps with our other ratios of 2-2.5x increase in size between buckets. It's a bit weird to add 25s, then only 15s when building a histogram.

Reply:
@csmarchbanks makes sense, updated as proposed